ple/games/donkeykong/OnBoard.py

```diff
@@ -0,0 +1,39 @@
+__author__ = 'Batchu Vishal'
+import pygame
+
+'''
+This class defines all inanimate objects that we need to display on our board.
+Any object that is on the board and not a person, comes under this class (ex. Coins,Ladders,Walls etc)
+Sets up the image and its position for all its child classes.
+'''
+
+
+class OnBoard(pygame.sprite.Sprite):
+    def __init__(self, raw_image, position, rng):
+        super(OnBoard, self).__init__()
+        self.__position = position
+        self.image = raw_image
+        self.rng = rng
```

Comment:
I also noticed that the `__init__` function of `OnBoard` accepts `rng` but you don't pass it to the super classes init method